src/DayHospital.java

```diff
@@ -20,44 +17,65 @@ public static void main(String[] args) {
 		String audit = "a";
 		int dayNo = 0;
 		
-		@SuppressWarnings("resource")
 		Scanner input = new Scanner(System.in);
 		System.out.println("Enter your input please: ");
 		
+		/**
+		 * A new instance of VetShop, petInterCare, is created
```

Comment:
If you are going to make a 1 line comment just use // otherwise you are just bloating the code

Reply:
changed to //